docs/examples.md

```diff
@@ -61,7 +61,7 @@ node -e "require('express')().use(require('express').static(__dirname, {index:'i
       },
       getFile(url) {
 
-        if ( url === '/main.vue' )
+        if ( url === './main.vue' )
```

Comment:
in fact, `if ( url === '/main.vue' )` is correct, but `loadModule('./main.vue', options)` is wrong, should be `loadModule('/main.vue', options)`. Feel free to update

Reply:
FYI, in `getFile(url)` the `url` argument should always refer to an absolute path.